.arm.docker-env.sample

```diff
@@ -0,0 +1,2 @@
+DOCKER_HOST="tcp://192.168.2.2:2375"
```

Comment:
Could you add a comment what this file does? It took a while before I understood the purpose, that it redirects docker commands on your machine to a Raspberry Pi